astropy/io/ascii/ui.py

```diff
@@ -660,9 +660,27 @@ def write(table, output=None,  format=None, Writer=None, fast_writer=True, **kwa
         List of names to exclude from output (applied after ``include_names``)
     fast_writer : bool
         Whether to use the fast Cython writer (default= ``True``)
+    overwrite : bool
+        Whether to overwrite an existing file (default= ``False``).
```

Comment:
The current default is ``None``.  Suggest this description of behavior:

```
If ``overwrite=None`` (default) and the file exists, then a warning
will be issued.  In a future release this will instead generate an exception.
If ``overwrite=False`` and the file exists, then an exception is raised.
```
The docstring should only document the current (and expected future) behavior.  Documenting changes from previous releases in the docstring frequently results in documentation rot.  I typically avoid making claims about future features in docs since they have a habit of not happening, but this one probably will and it's important to notify users.  But for back-compatibility issues the API changelog and What's New is the right place.